client/src/actions/index.js

```diff
@@ -211,10 +212,13 @@ const requestDifferentialExpression = (set1, set2, num_genes = 50) => async (
 
     const response = await res.json();
     const varIndex = await annoMatrix.fetch("var", varIndexName);
-    const data = response.map((v) => [
-      varIndex.at(v[0], varIndexName),
-      ...v.slice(1),
-    ]);
+    const data = { negative: [], positive: [] };
```

Comment:
Is there a more descriptive variable name that makes sense here, instead of `data`

Reply:
`data` is cxg's de facto name for action payloads

Reply:
Ah! Yes, referring to the private variable before assignment, so, perhaps:

```
return dispatch ({
  type: ...,
  data: _descriptivePrivateVariableName
})
```